mssql_rawquery_block.py

```diff
@@ -0,0 +1,50 @@
+from nio.properties import Property, VersionProperty, StringProperty
+from nio.block.terminals import output
+from .mssql_base import MSSQLBase
+from nio.block.mixins.enrich.enrich_signals import EnrichSignals
+
+@output('results', label='Results')
+@output('no_results', label='No Results')
+class MSSQLRawQuery(EnrichSignals, MSSQLBase):
+
+    version = VersionProperty("1.0.0")
+    query = StringProperty(title='Parameterized Query (use ? for any user-supplied values)', default='SELECT * FROM table where id=?', order=10)

```

Comment:
it might be better to use one of the alternate [paramstyle](https://www.python.org/dev/peps/pep-0249/#paramstyle) options: namely `numeric`. In the case where you want to reuse a single parameter multiple times:

```sql
SELECT baz FROM tbl where foo=:1 and bar=:1
```

Reply:
also here, is it possible to define a, what I called in the kotlin and javascript stacks, `StaticProperty`? A property that is guaranteed not to be interpolated and you just get the raw string value. Evaluating `_query = self.query(signal)` opens it up to the same signal/SQL injection issue, by-passing the parametric safety that you get with `.execute(query, params)`.

Reply:
I really thought we had a property argument called `allow_expr` that could be set to False for this kind of thing. The Retry mixin [actually uses it](https://github.com/niolabs/nio/blob/6326f08f59c33671dc522116562f3543d8396cca/nio/block/mixins/retry/retry.py#L21). But it doesn't seem to actually exist...